### ISSUE TITLE
Error getting access token: Response Error: 403 Account Temporarily Blocked

### ISSUE THREAD

**Issue by madegn** [matthillman/homebridge-tc-2-0#16]
After working for more than a year I'm getting the following message "Error getting access token: Response Error: 403 Account Temporarily Blocked"

This started about a week ago. If I go through the revalidation and reset of the password I'm able to use the plugin. But after a few hours the account ends up being temporarity blocked,

Has any one else seen this issue?

Thanks

**Comment by arlienuetzel** [comment]
I'm seeing the same error on my Homebridge. I wonder if they've started rate limiting and will eventually block access like liftmaster did with their garage door openers...

**Comment by madegn** [comment]
That was my thought also. I had my local alarm company out so that I could talk to Resideo directly. As expected not much help from them for a third party app.  Also tech support could not answer anything about Server Backend changes that may have caused these changes.

**Comment by arlienuetzel** [comment]
Indeed, when i went to reset that account password it cited a violation of the terms of service which included language about automatically connecting and controlling the hardware. Frustrating.

**Comment by madegn** [comment]
I'm guessing at this point not much we can do. I can got the Alarm/TotalConnectSecurity to function via voice command in Alexa but unable to integrate this into Apple Home app. I tried  to integrate with Alexa via a HOOBS server but not seeing support for a security systems with this integration into HOBBS/Homebridge functions .

**Comment by arlienuetzel** [comment]
Is there something in the architecture of the plugin that can be done to stay below the radar? Make the logons less frequent, more random, etc?

**Comment by madegn** [comment]
That is what I was wondering also but do not have the expertise to understand or modify that.

**Comment by platypus6** [comment]
The Home Assistant version of this app still works, which makes me think this app is the problem. My issue is neither this homebridge version or the HA version works with my alarm. I coded a custom plugin that worked flawlessly, and updated the HomeKit state properly. But then a few days later I was blocked due to violation of terms. So then I went to this app and tried it and same thing. So perhaps it's not my app either. But with the HA version working fine still as of Dec 2024, it makes me think a fix is possible.
